additional-appdata.patch

```diff
@@ -0,0 +1,52 @@
+diff --git a/pkg/desktop-appdata/speedcrunch.appdata.xml.in b/pkg/desktop-appdata/speedcrunch.appdata.xml.in
+index 3e89f973..37bc190a 100644
+--- a/pkg/desktop-appdata/speedcrunch.appdata.xml.in
++++ b/pkg/desktop-appdata/speedcrunch.appdata.xml.in
+@@ -5,6 +5,7 @@
+   <metadata_license>CC0-1.0</metadata_license>
+   <project_license>GPL-2.0+</project_license>
+   <update_contact>speedcrunch@googlegroups.com</update_contact>
++  <content_rating type="oars-1.0" />
+ 
+   <!-- TRANSLATORS: Name of the application, usually shouldn't be translated. -->
+   <_name>SpeedCrunch</_name>
+@@ -54,6 +55,13 @@
+   <url type="bugtracker">https://bitbucket.org/heldercorreia/speedcrunch/issues</url>
+   <url type="help">http://speedcrunch.org/introduction.html</url>
+ 
++  <releases>
++    <release version="0.12" date="2016-11-29"/>
++    <release version="0.11" date="2014-01-30"/>
++    <release version="0.10.1" date="2008-06-03"/>
++    <release version="0.10" date="2008-04-01"/>
++  </releases>
++
+   <!-- We can't use the <translations> tag since our qm files are baked into the
+        executable so we just use a static list instead. See:
+        https://blogs.gnome.org/hughsie/2016/01/25/appdata-and-the-gettext-domain/#comment-5137
+diff --git a/pkg/speedcrunch.appdata.xml b/pkg/speedcrunch.appdata.xml
```

Comment:
The appdata additions are upstream, yes ([link](https://bitbucket.org/heldercorreia/speedcrunch/commits/83760a3e978e8d84ed684367abb018b847caa0aa)). I don't see that changing the app id upstream has any advantages for us, though. I'd consider it if the `rename-*` steps didn't work out of the box.

Reply:
> I don't see that changing the app id upstream has any advantages for us, though. I'd consider it if the rename-* steps didn't work out of the box.

This will lead to duplicate entries in the GNOME Software, because we will have the same application with two IDs (one from repo, the other one from Flathub).
If it depended on me, I would leave it as it is. However, Flathub maintainers have repeatedly complained about it.

Reply:
@scx that issue was already fixed with a new entry that could be added to the desktop file. It's used when you have changed the app-ID to a correct one. Standards changes are for the good health of the Linux ecosystem. It will take a time to migrate and enforce some rules but it should happen.

Reply:
The id in the appdata you linked is just incorrect and I'm very confused why there are two.

Reply:
@TingPing Oh. I see. I wouldn't say "incorrect", I'd call it "outdated"; or alternatively, "vintage" ;) It looks like the file is based on an older version of the spec, maybe 0.10 or so. See roughly [here](https://github.com/ximion/appstream/blob/APPSTREAM_0_10_0/docs/sources/quickstart/desktop-apps.xml#L51): for desktop applications, the id was supposed to be the name of the desktop file, including the suffix as per the example. Which is how we did it.

I'll update it upstream for the next release. That being said, I checked the Flatpak locally and the result looks correct, so it's apparently not a problem yet.

As for why there are two, simple: one file is generated from the other by including the translations.